### PR TITLE
Pulp 1.6.8

### PR DESCRIPTION
**Pull request by johanneskoester**

### CODE REVIEW

File: recipes/pulp/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+source:
+  sha256: cfecd6a78c2b08539dc29b7e8281f43d729b1089bdb5e491c3c0956aadbb3fc7
+  fn: 'PuLP-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/P/PuLP/PuLP-{{ version }}.tar.gz
+
+build:
+  entry_points:
+    - pulptest=pulp:pulpTestAll
+    - pulpdoctest=pulp:pulpDoctest
```
Comment (isuruf) [inline recipes/pulp/meta.yaml]: Add spaces around `=`

File: recipes/pulp/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  number: 0
```
Comment (isuruf) [inline recipes/pulp/meta.yaml]: Should this package be `noarch: python`?

File: recipes/pulp/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - pyparsing >=2.0.1
```
Comment (dougalsutherland) [inline recipes/pulp/meta.yaml]: Don't think this is actually needed at build time.

File: recipes/pulp/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+about:
+  summary: |
+    PuLP is an LP modeler written in python. PuLP can generate MPS or LP files
+    and call GLPK, COIN CLP/CBC, CPLEX, and GUROBI to solve linear problems.
+  home: https://github.com/coin-or/pulp
+  license: MIT
+  doc_url: https://pythonhosted.org/PuLP/
+  dev_url: https://github.com/coin-or/pulp
+  license_family: MIT
```
Comment (dougalsutherland) [inline recipes/pulp/meta.yaml]: Add `license_file: LICENSE`, please.

File: recipes/pulp/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+
+extra:
+  recipe-maintainers:
+    - johanneskoester
+    - christopher-schroeder
```
Comment (dougalsutherland) [inline recipes/pulp/meta.yaml]: @christopher-schroeder, are you happy to [be a conda-forge maintainer](http://conda-forge.readthedocs.io/en/latest/recipe.html#maintainer-role) for the `pulp` package?

Comment (johanneskoester) [reply recipes/pulp/meta.yaml]: yes he is. We are sitting in the same office :-)

File: recipes/pulp/meta.yaml (hunk 4 above)
Comment (johanneskoester) [reply recipes/pulp/meta.yaml]: I don't think MIT requires this. But ok.

File: recipes/pulp/meta.yaml (hunk 3 above)
Comment (johanneskoester) [reply recipes/pulp/meta.yaml]: Yes, right. Good catch!

File: recipes/pulp/meta.yaml (hunk 2 above)
Comment (johanneskoester) [reply recipes/pulp/meta.yaml]: I tried, but it does not build in that case...

Comment (johanneskoester) [reply recipes/pulp/meta.yaml]: Seems that a platform specific binary for the lp-solving is used.

File: recipes/pulp/meta.yaml (hunk 1 above)
Comment (johanneskoester) [reply recipes/pulp/meta.yaml]: Done.

File: recipes/pulp/meta.yaml (hunk 5 above)
Comment (christopher-schroeder) [reply recipes/pulp/meta.yaml]: indeed :)

File: recipes/pulp/meta.yaml (hunk 4 above)
Comment (dougalsutherland) [reply recipes/pulp/meta.yaml]: MIT license says "The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software." I think source distributions definitely need it, and [it's ambiguous](https://github.com/github/choosealicense.com/issues/257) whether compiled versions do—but for Python code, you definitely have to include the license file.

Comment (johanneskoester) [reply recipes/pulp/meta.yaml]: Indeed, you are right I guess ;-).

**Pull request merged**